package.json

```diff
@@ -78,7 +78,7 @@
     "npm-run-all": "4.1.5",
     "prettier": "3.2.5",
     "ts-node": "10.9.2",
-    "typescript": "5.3.3"
+    "typescript": "5.6.2"
```

Comment:
Our linter complains for anything above 5.6.0, we should do something about this.

Reply:
Didn't notice, could you point out the error? I will have a look.

Reply:
Try running `yarn format`